application/features/VNC.py

```diff
@@ -41,16 +42,24 @@ def websocket_proxy_thread(local_websocket_port, local_vnc_port):
         #  1st: first handshake: upgrade the HTTP request
         #  2nd: actually serve the ws connection
         for _ in range(2):
+            if _ == 0:
```

Comment:
when an index is used, let's change its name to something non "_".

```suggestion
        for i in range(2):
            if i == 0:
```